pulp_ansible/app/tasks/copy.py

```diff
@@ -0,0 +1,69 @@
+from django.db import transaction
+from django.db.models import Q
+from pulpcore.plugin.models import RepositoryVersion
+
+from pulp_ansible.app.models import AnsibleCollectionDeprecated, AnsibleRepository, Collection
+
+
+@transaction.atomic
+def copy_content(config):
+    """
+    Copy content from one repo to another.
+
+    Accepts a config containing:
+      * source_repo_version_pk: repository version primary key to copy units from
+      * dest_repo_pk: repository primary key to copy units into
+      * content_pks: a list of content pks to copy from source to destination
```

Comment:
maybe `Accepts a config of a list of dictionaries containing:`

also the keys in the docstring seem to need editing, I needed to use:
* source_repo_version
* dest_repo
* content

Reply:
I will fix this, thank you

Reply:
Fixed, ty.